### ISSUE TITLE
Add support for indices_boost

### ISSUE THREAD

**Issue by insekticid** [ruflin/Elastica#1676]
https://www.elastic.co/guide/en/elasticsearch/reference/6.0/search-request-index-boost.html

```
GET /_search
{
    "indices_boost" : [
        { "alias1" : 1.4 },
        { "index*" : 1.3 }
    ]
}
```

solution:
$query = Query::create('food');
$query->setParam('indices_boost', ['recipe' => '1.4']);

**Comment by ruflin** [comment]
@insekticid Interested to contribute it? ;-)

**Comment by thePanz** [comment]
Please refer to the latest documentation: https://www.elastic.co/guide/en/elasticsearch/reference/master/search-request-body.html#request-body-search-index-boost :)

**Comment by deguif** [comment]
It was implemented in #2018
